package.json

```diff
@@ -18,16 +21,23 @@
   },
   "scripts": {
     "build": "npm run build-ottheme && npm run build-otkit",
-    "build-ottheme": "lerna exec --scope ottheme-* theo token.yml -- --transform web --format scss,cssmodules.css,common.js --dest .",
     "build-otkit": "lerna exec --scope otkit-* theo token.yml -- --transform web --format scss,cssmodules.css,common.js --dest .",
+    "build-ottheme": "lerna exec --scope ottheme-* theo token.yml -- --transform web --format scss,cssmodules.css,common.js --dest .",
     "clean": "lerna clean",
-    "test": "lerna exec --scope ot* theo token.yml -- --transform web --format scss,cssmodules.css,common.js",
+    "deploy-styleguide": "lerna exec --scope style-guide npm run deploy",
+    "develop": "concurrently 'lerna exec --scope style-guide npm run develop' 'npm-watch'",
     "postinstall": "lerna bootstrap",
-    "updatetoc": "doctoc README.md --github",
-    "publish": "npm run build && lerna publish --exact",
     "postpublish": "npm run deploy-styleguide",
+    "precommit": "lint-staged",
+    "publish": "npm run build && lerna publish --exact",
+    "test": "lerna exec --scope ot* theo token.yml -- --transform web --format scss,cssmodules.css,common.js",
     "updated": "lerna updated",
-    "develop": "concurrently 'lerna exec --scope style-guide npm run develop' 'npm-watch'",
-    "deploy-styleguide": "lerna exec --scope style-guide npm run deploy"
+    "updatetoc": "doctoc README.md --github"
+  },
+  "lint-staged": {
+    "*.js": [
+      "prettier --single-quote --write",
+      "git add"
```

Comment:
can we do the same for css?

Reply:
Yes for css, not sure for css modules, I'm investigating, thats why haven't added it there yet :)

Reply:
ok so I'll merge this then let's figure out css in a separate PR